### DISCUSSION TITLE
error, please help me

### DISCUSSION THREAD

**Discussion by Suz1e** [gildor2/UEViewer#274] [Q&A]
Unpack P U B G, but an error is reported, please help me
/Game/Weapon/Skin_2022/22Huya/Materials/DP12_22Huya_01.uasset
![QQ截图20220705151155](https://user-images.githubusercontent.com/52779408/177270865-ffccc201-1b24-45a9-a79e-c65672c2abd1.png)

**Comment by gildor2** [comment]
Check the forum, it has detailed information how to open PUBG models, possibly with custom UModel builds for this game.

**Reply by Suz1e** [reply]
I didn't find a solution, can you share it with me?
